src/components/HomeLogo.jsx

```diff
@@ -0,0 +1,12 @@
+import { Link } from "react-router";
+import homelogo from "../assets/northcodersnewslogosmalltransparent.png"
+
+export default function HomeLogo() {
+  return (
+    <div>
+      <Link to={"/"} className="home-logo-link">
+        <img src={homelogo}></img>
+      </Link>
+    </div>
+  );
```

Comment:
You don't really need a separate component for this. As it's part of the Header I'd put it in there.